frontend/src/modules/app/ToolBar.tsx

```diff
@@ -74,18 +80,35 @@ interface AlarmConfiguration {
   stateKey: string;
 }
 
+/**
+ * HeaderClock
+ *
+ * @component A container for displaying header clock of the page.
+ *
+ * @returns {JSX.Element} current time
+ */
 export const HeaderClock = (): JSX.Element => {
   const classes = useStyles();
   const clockTime = useSelector(getClockTime);
   return <span className={classes.paddingRight}>{clockTime}</span>;
 };
 
+/**
+ * PowerIndicator
+ *
+ * @component  A container for displaying battery indicator of the page.
```

Comment:
Does this just mean "A container for displaying the battery and power management status of the ventilator"? This isn't page-specific, right?

Reply:
Yes that's right. Its the container to display the battery and power management status of the ventilator. And yes it is not page specific. It is common for all pages